src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1433,7 +1433,16 @@ int32_t OpenRCT2::Scripting::GetTargetAPIVersion()
 {
     auto& scriptEngine = GetContext()->GetScriptEngine();
     auto& execInfo = scriptEngine.GetExecInfo();
-    return execInfo.GetCurrentPlugin()->GetTargetAPIVersion();
+
+    // Commands from the in-game console do not have a plug-in set
+    auto plugin = execInfo.GetCurrentPlugin();
+    if (!plugin)
```

Comment:
Looks like we're using the same pattern in other places, so I assume it works. I just can't find this being proper behaviour on cppreference.

Reply:
I think doing a nullptr check is perfectly fine. shared_ptr has specific overloads for ```operator ==``` and ```operator !=``` which accepts nullptr_t, so it will always use that.